sql-plugin/src/main/301until320-all/scala/com/nvidia/spark/rapids/shims/v2/TypeSigUtil.scala

```diff
@@ -16,12 +16,12 @@
 
 package com.nvidia.spark.rapids.shims.v2
 
-import com.nvidia.spark.rapids.{TypeEnum, TypeSig}
+import com.nvidia.spark.rapids.{TypeEnum, TypeSig, TypeSigUtilBase}
 
 import org.apache.spark.sql.types.DataType
 
 /** TypeSig Support for [3.0.1, 3.2.0) */
-object TypeSigUtil extends com.nvidia.spark.rapids.TypeSigUtil {
+object TypeSigUtil extends TypeSigUtilBase {
 
   /**
    * Check if this type of Spark-specific is supported by the plugin or not.
```

Comment:
Not caused by this PR, but while reviewing I noticed there's `allowDecimal` scaladoc params that are incorrect in this file.

Reply:
But this PR can help me to debug without any error.

Reply:
will check this para

Reply:
Done